examples/examples.js

```diff
@@ -65,11 +148,103 @@ const Random = {
             hexString: (digits) => {
                 const chars = '0123456789abcdef';
                 return this._seededRandomString(current, digits, chars);
+            },
+            
+            uuid: () => {
```

Comment:
why so much duplicate code? Can't the original function be written in a way that seed is an optional (last) argument?